lib/theme_check/shopify_liquid/object.rb

```diff
@@ -3,23 +3,19 @@
 
 module ThemeCheck
   module ShopifyLiquid
-    # TODO: (4/6) https://github.com/Shopify/theme-check/issues/656
-    # -
-    # Remove 'objects.yml' in favor of 'SourceIndex.objects'
-    # -
     module Object
       extend self
 
       def labels
-        @labels ||= YAML.load(File.read("#{__dir__}/../../../data/shopify_liquid/objects.yml"))
+        @labels ||= SourceIndex.objects.map(&:name)
       end
 
       def plus_labels
-        @plus_labels ||= YAML.load(File.read("#{__dir__}/../../../data/shopify_liquid/plus_objects.yml"))
+        @plus_labels ||= SourceIndex.plus_labels
```

Comment:
If the method is called `plus_labels` and does nothing more than returning the content of a file, the file should also be called `plus_LABELS` instead.